tools/run_tests/run_xds_tests.py

```diff
@@ -1087,11 +1087,9 @@ def patch_url_map_backend_service(gcp,
             'backendService': service.url,
             'weight': w,
         } for service, w in services_with_weights.items()]
-    } if services_withWeights else None
+    } if services_with_weights else None
 
     config = {
-        'defaultService':
```

Comment:
Isn't this necessary when backend_service is not None?

Reply:
This is patch, so the config here is the diff. This default service is set when creating the url_map, so it should be unnecessary to set it again.

Because hostname is specified, gRPC will always pick the path matcher specified below. Setting defaultService is noop anyway.

Reply:
Ah ok. I guess I just got used to expecting `urlMap.defaultService` and `urlMap.patchMatchers[0].defaultService` to always match (`test_change_backend_service` was intentionally updating both of these values). But if the path matcher is always used instead and the default service is irrelevant than this seems alright.